### DISCUSSION TITLE
can openucx be adapted to software programmed in go

### DISCUSSION THREAD

**Discussion by NUABO** [openucx/ucx#9511] [Q&A] [answered]
It is  convenient to call the openucx interface in a C++ program, but how do I need to call it in a Go language software? Thank you.

**Comment by ivankochin** [comment, marked as answer]
UCX has Go binding which wraps C interfaces to Go ones. Pls see here: https://github.com/openucx/ucx/tree/master/bindings/go

You can find the example of usage in the `bindings/go/tests` folder.

**Reply by NUABO** [reply]
thank you for your reply (:
